src/dash/parser/objectiron.js

```diff
@@ -32,37 +32,54 @@ import FactoryMaker from '../../core/FactoryMaker.js';
 
 function ObjectIron(mappers) {
 
-    function mergeValues(parentItem, childItem) {
-        for (let name in parentItem) {
-            if (!childItem.hasOwnProperty(name)) {
-                childItem[name] = parentItem[name];
+    function _conditionallyMapProperty(exception, parentName, parentIsArray, parentEl, child, mergeFlag) {
+        let allowMapping = true;
+
+        // check, if element matches an exception
+        if (exception) {
+            for (const [key, values] of Object.entries(exception)) {
```

Comment:
Suggest to put this into a private function `_mappingAllowed` and simply return if the function returns `false`